<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lophocphan</class>
 <widget class="QMainWindow" name="lophocphan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>661</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI Variable</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>931</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(89, 89, 89);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="windowNhanVien">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Sinh Viên</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="windowTinhLuong">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Tính Điểm</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnThoat">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>0</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Thoát</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="windowTinhLuong_2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>40</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Tính Điểm</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="windowChamCong_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>0</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Lớp Học Phần</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="windowChamCong">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Điểm Danh</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="txtTimKiem">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>811</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>nhập mã lớp học phần</string>
    </property>
    <property name="placeholderText">
     <string>Nhập tên nhân viên...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTimKiem">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>TÌM KIẾM</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblNhanVien">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>931</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Môn Học</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mã Lớp Hp</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thời Gian</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Học Kì </string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>931</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnThem">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>THÊM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSua">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>SỬA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnXoa">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>XÓA</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
